mlflow/resource_mlflow_model.go

```diff
@@ -50,6 +61,45 @@ func NewModelsAPI(ctx context.Context, m any) ModelsAPI {
 	return ModelsAPI{m.(*common.DatabricksClient), ctx}
 }
 
+func (a ModelsAPI) ListByFilter(filter string, order_by []string, page_token string) ([]Model, error) {
```

Comment:
please use Go SDK, and not implement this manually.

probably a good idea to migrate those mlflow resources to SDK first.

Reply:
https://pkg.go.dev/github.com/databricks/databricks-sdk-go@v0.5.0/service/mlflow#RegisteredModelsAPI.ListAll